File: SampleLibrary/SampleLibrary.Test/WordInfoTest.cs
using Xunit;

namespace SampleLibrary.Test
{
    public class WordInfoTest
    {
        [Fact]
        private void GetFileNameTest()
        {
            var wordInfo = new WordInfo("test", 20);
            Assert.Equal("test", wordInfo.GetFileName());
        }

        [Fact]
        private void GetPositionTest()
        {
            var wordInfo = new WordInfo("test2", 30);
            Assert.Equal(30, wordInfo.GetPosition());
        }
    }
}

File: Phase3/src/test/java/WordInfoTest.java
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class WordInfoTest {

    WordInfo wordInfo;
    @BeforeEach
    void initialize() {
        wordInfo = new WordInfo("test", 20);
    }

    @Test
    void getFileNameTest() {
        assertEquals("test", wordInfo.getFileName());
    }

    @Test
    void getPositionTest() {
        assertEquals(20, wordInfo.getPosition());
    }
}